### PR TITLE
Locale strings for the new progress tabs

### PR DESCRIPTION
**Pull request by kitallis**
**Story card:** https://www.pivotaltracker.com/story/show/171672661

## Because

The [new progress tab](https://github.com/simpledotorg/simple-server/pull/843) have a lot of strings that need to be localized.

## This addresses

* Left out all locales other an `en` as blank to be populated through Transifex later. 
* The new progress tab locales also sit under a different namespace: `user_analytics` so as to not affect the current progress tab in any way even if this PR gets deployed to Production.

### CODE REVIEW

File: config/locales/api/en.yml [hunk 1]
```diff
@@ -19,6 +19,22 @@ en:
...
+    hypertension: 'Hypertension'
+    diabetes: 'Diabetes'
+    genders: ['all', male', 'female', 'transgender']
+    all_time: 'All-time'
+    follow_up_patients: 'Follow-up hypertension patients'
+    registered_patients: 'Registered Patients'
+    hypertension_control: 'Hypertension control'
+    hypertension_control_desc: 'How many patients with hypertension had BP below 140/90 at their last follow-up visit in the month?'
+    notes: 'Notes'
+    footer_note_1: "Data is for all activity at %{facility_name}. If data appears out of date, tap the \"Sync pending\" link on the home screen."
```
Comment (harimohanraj89) [inline config/locales/api/en.yml]: Easier to wrap this in single quotes? I think the interpolation still works right?

File: config/locales/api/en.yml [hunk 2]
```diff
@@ -19,6 +19,22 @@ en:
     patients_enrolled: "How many new patients are enrolled at %{facility_name}?"
     all_time_patients: "All time: %{count} patients"
     updated_at: "Updated at %{time}"
+  user_analytics:
+    hypertension: 'Hypertension'
+    diabetes: 'Diabetes'
+    genders: ['all', male', 'female', 'transgender']
```
Comment (harimohanraj89) [inline config/locales/api/en.yml]: Will we be able to exclude `transgender` from, say, Ethiopia's progress tab?

**Comment by kitallis** [comment]
> When will these locales be _applied_ into the new progress tab view?

I think we'll just go back to https://github.com/simpledotorg/simple-server/pull/843 and read off of the locales. Pretty straightforward.

File: config/locales/api/en.yml (hunk 2 above)
Comment (kitallis) [reply config/locales/api/en.yml]: I don't exactly know how that will play out. But I believe the valid gender list will be a model level concern and thereby the presence of the gender translation should not be relevant to what data needs to be displayed.

Comment (harimohanraj89) [reply config/locales/api/en.yml]: Great point.

File: config/locales/api/en.yml (hunk 1 above)
Comment (kitallis) [reply config/locales/api/en.yml]: Resolved. It works!

**Comment by kitallis** [comment]
> Code changes look fine. Question: What happens if we merge this and #843 and a user views the progress tab in a non-English language? What will they see? Will their progress tab break?

Nope. Sorry, I mentioned this in the description. I've moved the new translations into a completely new locale namespace, so old things will not brea,

**Pull request merged**